velox/dwio/parquet/writer/Writer.h

```diff
@@ -36,7 +36,12 @@ struct WriterOptions {
   int32_t rowsInRowGroup = 10'000;
   int64_t maxRowGroupLength = 1'024 * 1'024;
   int64_t dictionaryPageSizeLimit = 1'024 * 1'024;
-  double bufferGrowRatio = 1;
+  // Parquet writer need to write the output into ArrowDataBufferSink.
```

Comment:
This comment can be streamlined to avoid repetition.

```
Growth ratio passed to ArrowDataBufferSink. The default value is a heuristic borrowed from folly/FBVector.
```

Reply:
It would be nice to include a link to https://github.com/facebook/folly/blob/main/folly/docs/FBVector.md#memory-handling which explains the choice of 1.5

Reply:
Updated.